ui/app/services/store.js

```diff
@@ -80,7 +80,7 @@ export default class StoreService extends Store {
       return resolve(this.fetchPage(modelName, query));
     }
     return adapter
-      .query(this, { modelName }, query)
+      .query(this, { modelName }, query, null, adapterOptions)
```

Comment:
curious what the null is for?

Reply:
[The docs](https://api.emberjs.com/ember-data/5.2/classes/RESTAdapter/methods/query?anchor=query) have it as `recordArray` but we aren't using it. Just needed to add it so I could pass along the `adapterOptions`.

Reply:
Ah, got it. Thank you for the docs link!